model/const.go

```diff
@@ -177,4 +177,10 @@ const (
 	MaxCallMsgSize = 25 * 1024 * 1024
 	// DefaultStreamBufSize defines gateway stream forward payload buf size
 	DefaultStreamBufSize = 16 * 1024 * 1024
+	// MaxRetryCount defines getting the latest height from the RPC client max retry count
+	MaxRetryCount = 50
```

Comment:
the const define please move to blocksyncer module dir, this dir will be deleted in later